apps/src/applab/designMode.js

```diff
@@ -34,6 +34,11 @@ designMode.onDivApplabClick = function (event) {
   if (element.id === 'divApplab') {
     designMode.clearProperties();
   } else {
+    if ($(element).is('.ui-resizable')) {
+      element = element.children[0];
+    } else if ($(element).is('.ui-resizable-handle')) {
+      element = element.parentNode.children[0];
+    }
```

Comment:
Looks like that works for the parent (ui-resizable) but not for the handle element. Might stick with using class name for consistency across both.